.github/workflows/build_android.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Android
+on:
+  workflow_dispatch:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+jobs:
+  Build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 12
+      - name: ktlint
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: ktlintCheck
+      - name: Lint
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: lint
```

Comment:
How can we see the Lint artifacts in the workflow results? Currently it doesn't seem to list them

Reply:
They need to be specifically retained. Not sure if that's necessary since we see the logs anyway.

Reply:
If it doesn't cost us anything, it'd be great to retain the Lint reports! The HTML reports are much easier to browse than spelunking the build logs.